shell/renderer/electron_renderer_client.cc

```diff
@@ -168,7 +168,7 @@ void ElectronRendererClient::WorkerScriptReadyForEvaluationOnWorkerThread(
   // that have a different value for nodeIntegrationInWorker
   if (base::CommandLine::ForCurrentProcess()->HasSwitch(
           switches::kNodeIntegrationInWorker)) {
-    WebWorkerObserver::GetCurrent()->WorkerScriptReadyForEvaluation(context);
+    WebWorkerObserver::Create()->WorkerScriptReadyForEvaluation(context);
```

Comment:
Maybe add a comment here to indicate `WorkerScriptReadyForEvaluationOnWorkerThread` can be invoked multiple times for the same thread and we create a new observer everytime when this happens. Additionally, `ThreadLocalOwendPointer` ensures that the old observer of this thread gets destructed when swapping with the new observer in `WebWorkerObserver::Create`